### PR TITLE
Remove start page in favor of a message

### PR DESCRIPTION
**Pull request by DonJayamanne**
For #6572

### CODE REVIEW

File: src/test/datascience/notebook/introStartPage.vscode.test.ts [hunk 1]
```diff
@@ -1,193 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (DonJayamanne) [inline src/test/datascience/notebook/introStartPage.vscode.test.ts:1]: I removed the tests, no point if this is going away and soon native notebook will be the default.

File: src/client/datascience/notebook/introStartPage.ts [hunk 2]
```diff
@@ -40,25 +36,26 @@ export class IntroduceNativeNotebookStartPage implements IExtensionSingleActivat
...
-        );
+    private notify() {
+        if (this.messageDisplayed) {
+            return;
+        }
+        this.messageDisplayed = true;
+        this.memento.update(IntroduceNativeNotebookDisplayed, true).then(noop, noop);
+        this.appShell
+            .showInformationMessage(
+                "Welcome to VS Code's new notebook experience!  We think you'll find it faster, prettier and easier to use! To learn more, click [here](https://github.com/microsoft/vscode-jupyter/wiki/Native-Notebook-Support-in-VS-Code)"
```
Comment (IanMatthewHuff) [inline src/client/datascience/notebook/introStartPage.ts:57]: This would need to be localized.

**Review by IanMatthewHuff — CHANGES_REQUESTED** [review]
Looks good, but the message needs to be in the loc file.

**Comment by DonJayamanne** [comment]
> Looks good, but the message needs to be in the loc file.

Does it, this is only temporary, isn't it.

File: package.nls.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
     "DataScience.insecureSessionDenied": "Denied connection to insecure server.",
     "DataScience.installPython": "Python is not installed. \nPlease [download](https://www.python.org/downloads) and install Python in order to execute cells in this notebook.",
     "DataScience.installKernel": "The Jupyter Kernel '{0}' could not be found and needs to be installed in order to execute cells in this notebook.",
+    "DataScience.newNotebookUI": "Welcome to VS Code's new notebook experience!  We think you'll find it faster and more pleasing to use! To learn more, click [here](https://aka.ms/NewNotebookUI)",
```
Comment (IanMatthewHuff) [inline package.nls.json:22]: My minor vote is this is two exclamation points too many. But I'll defer to @greazer and @claudiaregio on the text. Code look solid to me.

**Review by IanMatthewHuff — APPROVED** [review]
Approved per Lead / PM text approval.

**Pull request merged**